ee/clickhouse/models/property.py

```diff
@@ -419,6 +421,14 @@ def get_property_values_for_key(key: str, team: Team, value: Optional[str] = Non
     )
 
 
+def get_person_property_values_for_key(key: str, team: Team, value: Optional[str] = None):
+    if value:
+        return sync_execute(
```

Comment:
I'm just now learning there are people stored in Postgres 🤯

I think non-blocking because we have the same pattern elsewhere but you could collapse the two SQL queries into one if we kept `JSONHas(properties, %(key)s)` in both queries and templated in `trim(BOTH '\"' FROM JSONExtractRaw(properties, %(key)s)) ILIKE %(value)s` or an empty string

By having `trim(BOTH '\"' FROM JSONExtractRaw(properties, %(key)s))` hardcoded we can't take advantage of materialised columns either

Reply:
Since it's non blocking, then just in case I won't touch this to not make too many changes. This is regarding person properties though, so materialized columns aren't relevant here.

Reply:
Note materialized person properties/columns do exist.

Reply:
🤦